docs/Getting_Started.md

```diff
@@ -0,0 +1,437 @@
+
+To make development a bit simpler, the app uses [discord-interactions](https://github.com/discord/discord-interactions-js), which provides types and helper functions. If you prefer to use other languages or libraries, there’s [a page with community-built resources](#DOCS_TOPICS_COMMUNITY_RESOURCES) which you can browse through.
+
+### Remix the project
+
+This guide uses Glitch, which allows you to quickly clone and develop an app from within your browser. There are also instructions on developing locally using ngrok [in the README](https://github.com/shaydewael/discord-getting-started#running-app-locally) if you'd prefer.
+
+> info
+> While Glitch is great for development and testing, [it has technical limitations](https://help.glitch.com/kb/article/17-technical-restrictions/) so other hosting providers should be considered for production apps.
+
+To start, **[remix (or clone) the Glitch project 🎏](https://glitch.com/edit/#!/remix/getting-started-discord)**
+
+When you remix the project, you'll see a new Glitch project with a unique name similar to this:
+
+![Glitch project overview](glitch-project.png)
+
+#### Project structure
+
+All of the files for the project are on the left-hand side. Here's a quick glimpse at the structure:
+
+```
+├── examples    -> short, feature-specific sample apps
+│   ├── app.js  -> completed app.js code
+│   ├── button.js
+│   ├── command.js
+│   ├── modal.js
+│   ├── selectMenu.js
+├── .env        -> your credentials and IDs
+├── app.js      -> main entrypoint for app
+├── commands.js -> slash command payloads + helpers
+├── game.js     -> logic specific to RPS
+├── utils.js    -> utility functions and enums
+├── package.json
+├── README.md
+└── .gitignore
+```
+
+### Adding credentials
+
+There's already some code in your `app.js` file, but you’ll need your app’s token and ID to make requests. All of your credentials can be stored directly in the `.env` file.
+
+First, copy your bot user’s token from earlier and paste it in the **`DISCORD_TOKEN`** variable in your `.env` file.
+
+Next, navigate to your app settings in the developer portal and copy the **App ID** and **Public Key** from the **General Overview** page. Paste the values in your `.env` file as **`APP_ID`** and **`PUBLIC_KEY`**.
+
+Finally, fetch your guild ID by navigating to the server where you installed your app. Copy the first number in the URL after `channels/` (for example, in the URL `https://discord.com/channels/12345/678910`, the server ID would be `12345`). Save this value as **`GUILD_ID`** in your `.env` file.
+
+With your credentials configured, let's install and handle slash commands.
+
+### Installing slash commands
+
+> info
+> To install slash commands, the app is using [`node-fetch`](https://github.com/node-fetch/node-fetch). You can see the implementation for the installation in `utils.js` within the `DiscordRequest()` function. More information about Discord's REST API can be found in [the API reference](#DOCS_REFERENCE).
```

Comment:
Not to be that one guy, but is there a specific reasoning for using node-fetch and not undici fetch?